### PR TITLE
Update project to Qt5 and CMake 3.5; README update

### PR DESCRIPTION
**Pull request by Mappaschreck**
- Upgraded project from Qt4 to Qt5.
- Set minimum CMake version to 3.5.
- Updated README to recommend 'libusb-dev' instead of 'libusb'.

### CODE REVIEW

File: ds9490.cpp [hunk 1]
```diff
@@ -76,8 +77,8 @@ bool DS9490::AquireUsb(struct usb_device* dev)
       m_lastError = "Failed to open USB device";
       return false;
    }
-   if (usb_set_configuration(m_usbDevHandle, 1)!=0)
-   {
+   if (usb_set_configuration(m_usbDevHandle, 1) != 0) {
+      std::cerr << "usb_set_configuration error: " << usb_strerror() << std::endl;
```
Comment (kkoop) [inline ds9490.cpp]: I would prefer not directly outputting to stderr from this class, instead putting the error message into m_lastError which can then be either printed to stderr or into a QMessageBox by the caller

File: gui/mainwindow.cpp [hunk 2]
```diff
@@ -138,8 +138,8 @@ void MainWindow::onReadData()
    
    for(int i=0; i<sampleCount; i++){
       QDateTime addedTime=timeStamp.addSecs(i*sampleRate);
-      QString dateLocaleTime = addedTime.date().toString(Qt::SystemLocaleShortDate)
-                              +" "+addedTime.time().toString(Qt::TextDate);      
+ QString dateLocaleTime = QLocale().toString(addedTime.date(), QLocale::ShortFormat)
```
Comment (kkoop) [inline gui/mainwindow.cpp]: indentation looks off here

Comment (Mappaschreck) [reply gui/mainwindow.cpp]: Thank you for pointing this out; I have corrected the passage.

File: ds9490.cpp (hunk 1 above)
Comment (Mappaschreck) [reply ds9490.cpp]: Thanks for the feedback! I have adjusted the error handling and added additional information about usb_strerror().

**Pull request merged**